qiita_db/analysis.py

```diff
@@ -160,6 +162,22 @@ def create(cls, owner, name, description, parent=None, from_default=False):
             conn_handler.add_to_queue(
                 queue, sql, (owner.id, name, description, status_id))
 
+        # make changes to SQL needed to add to both QIITA and given portal
+        insert_portal = ""
```

Comment:
Ok, in such case add the returning to the above `INSERT`, but this code needs to be refactored anyways.